lib/rspecq/reporter.rb

```diff
@@ -90,5 +103,20 @@ def failure_formatted(rspec_output)
     def humanize_duration(seconds)
       Time.at(seconds).utc.strftime("%H:%M:%S")
     end
+
+    def flaky_jobs_to_sentry(jobs, build_duration)
+      return if jobs.empty?
+
+      Raven.capture_message("Flaky jobs detected", level: "warning", extra: {
```

Comment:
So I thought about that a bit. The issue with having each flaky job as a separate event is that, across builds, a given job (e.g. `foo_spec.rb[1:2:3]`) might refer to _different_ examples (since examples are added, changed, deleted). So the grouping Sentry will actually be misleading. Also, I'm afraid a bit about cases where there are thousands of failures, so I'd wait for #19 to be implemented before doing something like this.

However having the spec file in the fingerprint and the tag as you suggest, sounds pretty handy. We could do that in a subsequent iteration, after we gather some feedback from this implementation.

Reply:
Sounds reasonable. I did not think of the spec file structure. :+1: